Includes/shared_fields_common.view.lkml

```diff
@@ -73,30 +73,15 @@ view: shared_fields_common {
     group_label: "Browser"
   }
 
-  # browser_view_height is not in Sessions
 
-  dimension: browser_view_height {
-    description: "The browser viewport height in pixels."
-    type: number
-    sql: ${TABLE}.br_viewheight ;;
-    group_label: "Browser"
-  }
-
-  # browser_view_width is not in Sessions
-  dimension: browser_view_width {
-    description: "The browser viewport width in pixels."
-    type: number
-    sql: ${TABLE}.br_viewwidth ;;
-    group_label: "Browser"
-  }
 
 # Location
 
   dimension: geo_country {
     type: string
     sql: ${TABLE}.geo_country ;;
     group_label: "Location"
-    drill_fields: [page_display_url]
+    drill_fields: [shared_fields_no_session.page_display_url]
```

Comment:
This doesn't work right. There is no Drill Down on the explore: https://analytics.gov.bc.ca/explore/snowplow_web_block/clicks?qid=ArrBDLrZEraDdtwYfGSb77&toggle=fil

In a test branch I tried replacing this with:
```
drill_fields: [page_display_url]
```
That causes a Looker Error, but when while it doesn't let me commit with errors, I can still test the outcomes in an explore, and the explore linked to above works correctly when I do (it shows the drilldown expected).

We don't want to commit code with syntax errors (even though technically it is working, and through administrative privilege we _can_ commit error-containing code)


Instead, much like you do in `sessions.view`, I think we will have to override this `geo_country` dimension in clicks, page views, and searches to contain:
```
drill_fields: [page_display_url]
```
and then remove the line
```
     drill_fields: [shared_fields_no_session.page_display_url]
```
from this `shared_fields_common` file.

Slightly less DRY, but better than committing code that Looker's syntax tree doesn't parse without throwing an error.